proxy-login-automator.js

```diff
@@ -167,7 +167,7 @@ function createPacServer(local_host, local_port, remote_host, remote_port, buf_p
             }).on('end', function () {
                 var s = Buffer.concat(buf_ary).toString();
                 buf_ary = [];
-                s = s.replace(/(['"])PROXY\s+([^'":\s]+)(:\d+)?\1/g, function (matched_all, matched_quot, matched_remote_host, matched_comma_remote_port) {
+                s = s.replace(/PROXY\s+([^'":\s]+)(:\d+)?/g, function (matched_all, matched_remote_host, matched_comma_remote_port) {
```

Comment:
Thank you!
Do you means PAC like the following ?
```
return "PROXY 4.5.6.7:8080; PROXY 7.8.9.10:8080";
```
It seems your regex still can not does not replace second host:port after ';'.

So i further modified it to :
```
s.replace(/(['";]?\s*PROXY\s+)([^:]+):(\d+)(\s*['";])/g, function (matched_all, head, host, port, tail) {
        return head+'127.0.0.1:9999'+tail; //just as a sample
    })
```

Some simple test:
```
function replace(s) {
    console.log(s.replace(/(['";]?\s*PROXY\s+)([^:]+):(\d+)(\s*['";])/g, function (matched_all, head, host, port, tail) {
        return head+'127.0.0.1:9999'+tail;
    }));
}

replace('return "PROXY a.b.c:8080"');
replace("return 'PROXY a.b.c:8080'");
replace('return "PROXY a.b.c:8080; d.e.f:8080"');
replace("return 'PROXY a.b.c:8080; d.e.f:8080'");
replace('return "PROXY a.b.c:8080;"');
replace('return " PROXY  a.b.c:8080 "');
```

Result:
```
> replace('return "PROXY a.b.c:8080"');
return "PROXY 127.0.0.1:9999"
> replace("return 'PROXY a.b.c:8080'");
return 'PROXY 127.0.0.1:9999'
> replace('return "PROXY a.b.c:8080; d.e.f:8080"');
return "PROXY 127.0.0.1:9999; d.e.f:8080"
> replace("return 'PROXY a.b.c:8080; d.e.f:8080'");
return 'PROXY 127.0.0.1:9999; d.e.f:8080'
> replace('return "PROXY a.b.c:8080;"');
return "PROXY 127.0.0.1:9999;"
> replace('return " PROXY  a.b.c:8080 "');
return " PROXY  127.0.0.1:9999 "
```